### PR TITLE
Add WCA Competition Tutorial to the website.

### PR DESCRIPTION
**Pull request by jfly**
Thanks to the WQAC for putting this together!

image

### CODE REVIEW

File: WcaOnRails/config/routes.rb [hunk 1]
```diff
@@ -160,7 +161,7 @@
   end
 
   namespace :api do
-    get '/', to: redirect('/api/v0')
+    get '/', to: redirect('/api/v0', status: 302)
```
[the remark this answers is not in the document]
Comment (jfly) [reply WcaOnRails/config/routes.rb]: Nope, I just realized while I was in there that we should be using a temporary redirect here. I can pull this out into a separate PR if you like.

**Comment by viroulep** [comment]
Looks like travis is unhappy: we test the locale switcher by switching to French, and it means that all locale keys used during this process must be defined :(
@jfly: I think adding the French translation for the `.tutorial` would be enough, could you please add it?
It's "Tutoriel pour les compétitions".

**Comment by suushiemaniac** [comment]
I'd like to see this PR get merged as soon as possible. Is there something I can do other than nagging software people? :)

**Comment by jfly** [comment]
Thanks for doing the investigation, @viroulep! I've added that translation for French just to get the build happy. I will merge this up as soon as travis passes.

**Pull request merged**